INTHEWILD.md

```diff
@@ -386,3 +386,4 @@ Currently **officially** using Airflow:
 1. [Zerodha](https://zerodha.com/) [[@johnnybravo-xyz](https://github.com/johnnybravo-xyz)]
 1. [Zymergen](https://www.zymergen.com/)
 1. [Zynga](https://www.zynga.com)
+1. [Graduate School, University of Southern California (USC)](https://graduateschool.usc.edu/) [[@abhilash1in](https://github.com/abhilash1in), [@sudarshansunder](https://github.com/sudarshansunder)]
```

Comment:
Thanks for the PR, can you add while preserving alphabetical order, please?

Reply:
Apologies, didn't notice the ordering. I'll update the file. Thanks!